packages/app/src/app/overmind/namespaces/editor/internalActions.ts

```diff
@@ -40,7 +40,7 @@ export const initializeSandbox = async (
   sandbox: Sandbox
 ) => {
   await Promise.all([
-    actions.editor.internal.initializeLiveSandbox(sandbox),
+    // actions.editor.internal.initializeLiveSandbox(sandbox),
```

Comment:
@tristandubbeld I left it commented because I'm waiting for @necoline's answer here: https://github.com/codesandbox/codesandbox-client/pull/7413#discussion_r1099637570. Will restore/delete the line after a get a reply :)

Reply:
Removed the comment here: [68292fb](https://github.com/codesandbox/codesandbox-client/pull/7413/commits/68292fb2c342029c7e3a4c62c5322a6f9cb5e367)